yb-voyager/src/dbzm/dbzm.go

```diff
@@ -33,7 +33,7 @@ import (
 var DEBEZIUM_DIST_DIR, DEBEZIUM_CONF_FILEPATH string
 
 // These versions need to be changed at the time of a release
-const DEBEZIUM_VERSION = "2.2.0-1.3.0"
+const DEBEZIUM_VERSION = "2.2.0-1.6.5-rc1"
```

Comment:
@ShivanshGahlot we also updated dbzm version, so let's call this 
`2.3.3-1.6.5-rc1`